Doppler.BillingUser/Authorization/IUsersApiTokenGetter.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Doppler.BillingUser.Authorization
+{
+    public interface IUsersApiTokenGetter
```

Comment:
Por lo que vi el token que se desea obtener es para la API https://apisint.fromdoppler.net/doppler-account-plans. Si esto es correcto, qué te parece lo siguiente?
```suggestion
    public interface IAccountPlansApiTokenGetter
```